generator.go

```diff
@@ -30,7 +30,7 @@ type Example struct {
 	value interface{}
 }
 
-func (g *DataGenerator) generateInternal(schema *spec.Schema, requestPath string, expansions *ExpansionLevel, example *Example, context string) (interface{}, error) {
+func (g *DataGenerator) generateInternal(schema *spec.Schema, requestPath string, id *string, replacedID *string, doReplaceID bool, expansions *ExpansionLevel, example *Example, context string) (interface{}, error) {
```

Comment:
Not super happy about continued argument proliferation here. I think we need to change this to an args struct, but I want to do that on a different PR.